### DISCUSSION TITLE
creating data base for trainin ct angio pictures

### DISCUSSION THREAD

**Discussion by esikl** [voxelmorph/voxelmorph#475] [Q&A]
hi everyone, i want to create a data base for training voxelmorph network for registration of ct/angio pictures
i used 30 dicom pictures and read each frame as a picture.
then i have 30 folders which every folder includes images related to different frames , for example if a dicom consists  of 15 frames, there are 15 images in the folder which have  movement(because heart movement)
i want to registers all of images related to each dicom.
please tell me how prepare database to train network for doing this job
thanks

**Comment by adalca** [comment]
I don't fully understand how you have these data setup, perhaps a folder/image tree would help?

@ahoopes is there a variant in trainig where we can supply a text file with *pairs* of images on each row?

**Reply by esikl** [reply]
thanks for collaborating, i don't have any idea i just want to start, i want to know if these images are useful for training?
and if yes how can i make related images and labels?

**Reply by esikl** [reply]
for example in my mined, i want to labels all the same images( different frames of a dicom file) as same label (for example 1) and there for for each folder  a labels will be exist, and in this way there are about 30 classes and each class has between 12 to 20 members( according to number of dicom file frames)
